config/deploy/master.rb

```diff
@@ -1 +1,2 @@
 set :branch, 'master'
+set :vhost, 'master.hacken.in'
```

Comment:
Should we maybe do the same for the port? But we can't read it from the .htaccesss ,right?